.gitignore

```diff
@@ -4,4 +4,5 @@
 *.js
 *.d.ts
 *.js.map
-debug.log
\ No newline at end of file
+debug.log
+/.vs
```

Comment:
what for remove /.vs directory? we can store there project defined configs for VS Code